mayhem/fuzz_parse.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python3
```

Comment:
In the future a slightly more explicitly filename, like `fuzz_parse_fromstring`, would be preferable.